src/App/Order/Calculator/PostNl/PostNLReceiptCodeCalculator.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\App\Order\Calculator\PostNl;
+
+use MyParcelNL\Pdk\App\Order\Calculator\AbstractPdkOrderOptionCalculator;
+use MyParcelNL\Pdk\Base\Service\CountryCodes;
+use MyParcelNL\Pdk\Types\Service\TriStateService;
+use MyParcelNL\Pdk\Validation\Validator\CarrierSchema;
+use MyParcelNL\Pdk\Facade\Pdk;
+
+/**
+ * When receipt code is enabled, insurance is required.
+ */
+final class PostNLReceiptCodeCalculator extends AbstractPdkOrderOptionCalculator
+{
+    public function calculate(): void
```

Comment:
Deze functie mist nog een docblock.

Ik denk sowieso wel fijn om overal een functionele omschrijving / description in de PHPdoc blocks te gaan zetten, maar dat doen we tot nu toe nog niet altijd, dus laat ik even aan jou of je dat hier ook wil doorvoeren.

Wel fijn om de autogenerated docs er consistent bij te zetten tho.